lib/std/fs.zig

```diff
@@ -221,7 +221,7 @@ pub fn renameZ(old_dir: Dir, old_sub_path_z: [*:0]const u8, new_dir: Dir, new_su
 /// Same as `rename` except the parameters are WTF16LE, NT prefixed.
 /// This function is Windows-only.
 pub fn renameW(old_dir: Dir, old_sub_path_w: []const u16, new_dir: Dir, new_sub_path_w: []const u16) !void {
-    return posix.renameatW(old_dir.fd, old_sub_path_w, new_dir.fd, new_sub_path_w);
+    return posix.renameatW(old_dir.fd, old_sub_path_w, new_dir.fd, new_sub_path_w, windows.FALSE);
```

Comment:
```suggestion
    return posix.renameatW(old_dir.fd, old_sub_path_w, new_dir.fd, new_sub_path_w, windows.TRUE);
```

? `FALSE` doesn't seem to match [the expected `rename()` semantics](https://github.com/ziglang/zig/blob/a5207190eb5d88a6d7b0c71c93b2593c5ebf77ce/lib/std/fs/Dir.zig#L1748).

Reply:
> ? `FALSE` doesn't seem to match [the expected `rename()` semantics](https://github.com/ziglang/zig/blob/a5207190eb5d88a6d7b0c71c93b2593c5ebf77ce/lib/std/fs/Dir.zig#L1748).

Right, I wasn't sure what would be the appropriate default for that. Will update it.